test/lib/utils/backup_test.rb

```diff
@@ -38,6 +38,10 @@ module ForemanMaintain
       file_path = '../../files/backups/katello_logical_pulp2_pulpcore_database'
       File.expand_path(file_path, File.dirname(__FILE__))
     end
+    let(:katello_hybrid_db_backup) do
+      file_path = '../../files/backups/katello_hybrid_db_backup/'
```

Comment:
Probably best for a separate PR, but have you thought about a `fixture()` helper method so it would be `let(:katello_hybrid_db_backup) { fixture('backups/katello_hybrid_db_backup') }`

Reply:
+1 https://projects.theforeman.org/issues/33588